src/Database/TypeFactory.php

```diff
@@ -44,6 +44,7 @@ class TypeFactory
         'float' => Type\FloatType::class,
         'json' => Type\JsonType::class,
         'string' => Type\StringType::class,
+        'char' => Type\CharType::class,
```

Comment:
`CharType` is identical to `StringType` isn't it? If so then I don't see the need for code duplication. The `char` type can just be mapped to `StringType` similar to `text`.

Reply:
Nope.
Since 7.2 The flag is "PDO::PARAM_STR_CHAR" Ref:  https://www.php.net/manual/en/pdo.constants.php 
However, I was not sure of the compatibility i.e. what is the minimum php we need for 4x so I kept it as "PDO::PARAM_STR".
Created a new Type so that once we deprecate support for < 7.2, we can use the new flag.
Turns out, Travis is set to minimum 7.2 , should I go ahead and put the new flag instead? What do you advise?

Reply:
CakPHP required PHP 7.2+ so that's not an issue. The description for `PDO::PARAM_STR` says `Represents the SQL CHAR, VARCHAR, or other string data type.` so it seems good enough for `char` fields too.

For `PDO::PARAM_STR_CHAR` it says `Flag to denote a string uses the regular character set.`. I am not sure what that means and in what case it should be used. Maybe someone more knowledge on that matter does.

So for now I suggest we simply use the `StringType` class. We can always update later if needed.

Reply:
Cool! On it!